gradio/routes.py

```diff
@@ -302,7 +304,7 @@ async def predict(
                 if body.fn_index is None:
                     return JSONResponse(
                         content={
-                            "error": f"This app has no endpoint /api/{api_name}/."
+                            "error": f"This app has no endpoint /run/{api_name}/."
```

Comment:
It looks like we are migrating from "/api" to "/run" as the official API endpoint. Let's make sure to update the "view API" page as well.

Also this is a bit of an edge case but what happens if a user-supplied `api_name` includes the string "api". Perhaps we can print a warning to users letting them know that their demos will not work inside Colab

Reply:
Does colab block any call with a substring "api"? Anyways, even if they create a custom named API, the frontend does not use the names API's, so gradio UI will still work fine.